java-security/src/main/java/com/sap/cloud/security/config/k8s/K8sEnvironment.java

```diff
@@ -14,76 +22,120 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.*;
-import java.util.function.UnaryOperator;
 import java.util.stream.Collectors;
 
 import static com.sap.cloud.security.config.Service.IAS;
 import static com.sap.cloud.security.config.Service.XSUAA;
+import static com.sap.cloud.security.config.k8s.K8sConstants.*;
 
 /**
  * Loads the OAuth configuration ({@link OAuth2ServiceConfiguration}) of a
  * supported identity {@link Service} in the Kubernetes Environment by
- * accessing defaults service secrets paths "/etc/secrets/sapcp/xsuaa" for Xsuaa service or "/etc/secrets/sapcp/ias" for .
- * IAS service
+ * accessing defaults service secrets paths "/etc/secrets/sapcp/xsuaa" for Xsuaa service
+ * or "/etc/secrets/sapcp/ias" for IAS service.
  */
 public class K8sEnvironment implements Environment {
```

Comment:
I dislike this class, as it is huge with several responsibilities and contains more or less only static methods which are quite hard to test. I think we can split it up into some minor classes, one may respresent the serviceConfigurations loaded for example.